tests/column_resizer_tester.ts

```diff
@@ -0,0 +1,264 @@
+import { Expect, Test } from 'alsatian';
+import { ColumnResizer, TableInterface } from '../source';
+
+class FauxTableInterface implements TableInterface {
+  constructor() {
+    this._columnCount = 3;
+    this._activeWidth = 20;
+    this._widths = [200, 300, 200];
+    this._corners = {topLeft: {x: 0,y: 0}, bottomRight: {x: 700,y: 100}};
+  }
+
+  public get columnCount() {
+    return this._columnCount;
+  }
+
+  public get activeWidth() {
+    return this._activeWidth;
+   }
+
+  public get corners() {
+    return this._corners;
+  }
+
+  public getColumnWidth(index: number) {
+    return this._widths[index];
+  }
+
+  public onResize (columnIndex: number, difference: number){
+    if(columnIndex >= this._columnCount) {
+      throw RangeError;
+    }
+    if(difference === 0){
+      return;
+    }
+    const minWidth = 20;
+    const changedWidth = this._widths[columnIndex] + difference;
+    if(changedWidth < minWidth) {
+      this._widths[columnIndex] = minWidth;
+    } else {
+      this._widths[columnIndex] = changedWidth;
+    }
+    this._corners.bottomRight.x = 0;
+    for(let i = 0; i < this._columnCount; ++i) {
+      this._corners.bottomRight.x += this._widths[i];
+    }
+  }
+
+  private _columnCount: number;
+  private _activeWidth: number;
+  private _corners: 
+    {topLeft: {x: number, y: number}, bottomRight: {x: number, y: number}};
+  private _widths: number[];
+}
+
+class MouseEvent {
+  constructor(cx: number, cy:number, mx: number){
+    this._clientX = cx;
+    this._clientY = cy;
+    this._movementX = mx;
+  }
+
+  public get clientX() {
+    return this._clientX;
+  }
+
+  public get clientY() {
+    return this._clientY;
+  }
+
+  public get movementX() {
+    return this._movementX;
+  }
+
+  private _clientX: number;
+  private _clientY: number;
+  private _movementX: number;
+}
+
+/** Tests the ColumnResizer. */
+export class ColumnResizeTester {
+
+  /** Tests what happens when a mousedown happens in the resize region
+   * and moved in a positive direction.
```

Comment:
Align documentation.

Reply:
Still needs to be resolved.

Reply:
What do you mean by align? As in put spaces on line 83 until it starts right under "Tests"?

Reply:
Yes.